Rakefile

```diff
@@ -1,5 +1,14 @@
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new
+
 task default: %w[test]
 
 task :test do
-  sh "bundle exec rspec"
+  sh 'bundle exec rspec'
 end
+
+task test: :rubocop
+
+# task :rubocop do
+#   sh 'rubocop'
+# end
```

Comment:
If this rake task is not used, delete it rather than commenting it out.

Reply:
I will delete it, I was confirming that the script I wrote from the rubocop docs worked before deleting the first one I found.